designer/app.py

```diff
@@ -760,12 +761,8 @@ def _perform_new(self, *args):
 
         self.close_popup()
 
-        new_proj_dir = os.path.join(get_config_dir(),
-                                    constants.NEW_PROJECT_DIR_NAME)
-        if os.path.exists(new_proj_dir):
-            shutil.rmtree(new_proj_dir)
-
-        os.mkdir(new_proj_dir)
+        new_proj_dir = tempfile.mkdtemp(prefix='designer_',
+                                        dir=get_config_dir())
```

Comment:
Looks like that the option 1 is better here, let's move to the @MysticMight's PR. I'd rather to let the OS handle this issue (it's a better solution, and easier to maintain), however, if Windows is not doing it in the right way, the better option is to delete it. 

The big problem: if you build your new project using buildozer, for example, this "temp" folder will have some megabytes. And if the user keep doing this, creating a new project, testing something and building, KD can easily creates GBs of data in the temp folder.